cmd/terramate/cli/cli.go

```diff
@@ -597,14 +604,59 @@ func (c *cli) cloneStack() {
 }
 
 func (c *cli) generate(workdir string) {
-	report := generate.Do(c.cfg(), workdir)
+	report, vendorReport := c.gencodeWithVendor(workdir)
```

Comment:
Hmm maybe I missed something, but at first I can't merge them, they are separated exactly because we handle the codegenReport + vendorReport differently on stack creation, so I can't have a single function that does everything (which would be more ideal).

I do agree that the name is bad...really struggling with it, because yeah vendor is always done, it is just a matter of having default report handling or customized report handling.

Reply:
> they are separated exactly because we handle the codegenReport + vendorReport differently on stack creation

When I added this review the stack creation handling was not implemented yet... I thought they would be the same.

Reply:
Ah all good :+1: